modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -820,8 +816,49 @@
 		end
 	end
 
+	function dotnetbase.netcore.getsdk(cfg)
+        local map = {
+			["Default"] = "Microsoft.NET.Sdk",
+            ["Web"] = "Microsoft.NET.Sdk.Web",
+            ["Razor"] = "Microsoft.NET.Sdk.Razor",
+            ["Worker"] = "Microsoft.NET.Sdk.Worker",
+            ["Blazor"] = "Microsoft.NET.Sdk.BlazorWebAssembly",
+            ["WindowsDesktop"] = "Microsoft.NET.Sdk.WindowsDesktop",
+            ["MSTest"] = "MSTest.Sdk",
+        }
+
+		if cfg.flags.WPF then
+			return map["WindowsDesktop"]
+        end
+
+        return map[cfg.dotnetsdk or "Default"]
+	end
+
+	function dotnetbase.netcore.dotnetsdk(cfg)
+		dotnetbase.generate_global_json(cfg)
+	end
+
 	function dotnetbase.allowUnsafeBlocks(cfg)
 		if cfg.clr == "Unsafe" then
 			_p(2,'<AllowUnsafeBlocks>true</AllowUnsafeBlocks>')
 		end
 	end
+
+	function dotnetbase.output_global_json(prj) -- unsure how you handle project with different config
```

Comment:
The comment was for you ;-)
Does it make sense that user have something like

```
filter "configuration:Release"
    dotnetsdk "Web"
filter "configuration:Test"
    dotnetsdk "MSTest"
```
How do you handle it?
As it seems global.json would be ignored with other `dotnetsdk`, checking that at least one cfg has this flag might be the solution

Reply:
no not really as they are seperate frameworks i see absolutly no reason ass it imports different namespaces etc and would break code!